security.md

```diff
@@ -25,12 +23,12 @@ When using delegated authentication or service account authentication, Simeon us
 
 ### Approvals
 
-Before making any changes to your tenant, Azure DevOps will wait for you to explicitly approve the changes to be made.
+Before making any changes to your tenant, Azure DevOps will wait for you to explicitly approve the changes to be made. Specific users or groups can be granted access to approve changes to ensure control over this process.
```

Comment:
I am wondering if we can get the same value out of this sentence with fewer words. No worries if you prefer not to take the below change

```suggestion
Before making any changes to your tenant, Azure DevOps will wait for you to explicitly approve the changes to be made. Specific users or groups can be granted access to approve changes.
```